rodas2021rehamiltonian/openreview/packages.tex

```diff
@@ -0,0 +1,15 @@
+\usepackage[]{neurips_2019}
+\usepackage{makecell}
+\usepackage{tabularx}
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```

Comment:
Please remove the fontenc import as well (check #4 for discussions)

Reply:
Fixed in commit dcea8addb4e018c8b16534d83a57b3a851929ada